webpack.config.extensions.js

```diff
@@ -56,14 +56,17 @@ const editorBetaScript = [
 	...blockScripts( 'editor', path.join( __dirname, 'extensions' ), allPresetBlocks ),
 ];
 
-const webpackConfig = getBaseWebpackConfig( null, {
-	entry: {
-		editor: editorScript,
-		'editor-beta': editorBetaScript,
-		...viewBlocksScripts,
-	},
-	'output-path': path.join( __dirname, '_inc', 'blocks' ),
-} );
+const webpackConfig = getBaseWebpackConfig(
+	{ WP: true },
```

Comment:
Should we leave a comment above this line what this signifies?

Reply:
IMO that's part of that functions's API, so above all, it should be documented in its JSDoc (rather than at callsites). But yeah, we can also add a comment here. There's a chance though that it'll go stale if we say something like _Used to indicate that we're externalising `@wordpress/` dependencies_ -- if we decide e.g. that option should mean other things as well (using the `wordpress-element` Babel preset comes to mind.

Reply:
Valid points for not adding it. In this kind of big long-living repositories information becoming stale is a real issue.